src/w3iProxy/w3iChatFacade.ts

```diff
@@ -11,9 +11,19 @@ import type {
 } from './chatProviders/types'
 import InternalChatProvider from './chatProviders/internalChatProvider'
 import ExternalChatProvider from './chatProviders/externalChatProvider'
+import { distinct, filter, from, ReplaySubject, throwError, timeout } from 'rxjs'
+// eslint-disable-next-line no-duplicate-imports
 import { fromEvent } from 'rxjs'
```

Comment:
Do we need the two distinct import statements here since it's both just `import`, not `import` + `import type`?

Reply:
Might as well disable this lint rule anyways since it keeps flagging `import` + `import type` which is widely used in the w3i codebase